mutation.hh

```diff
@@ -206,7 +206,7 @@ namespace {
 template<consume_in_reverse reverse, FlattenedConsumerV2 Consumer>
 std::optional<stop_iteration> consume_clustering_fragments(schema_ptr s, mutation_partition& partition, Consumer& consumer, mutation_consume_cookie& cookie, is_preemptible preempt = is_preemptible::no) {
     constexpr bool crs_in_reverse = reverse == consume_in_reverse::legacy_half_reverse || reverse == consume_in_reverse::yes;
-    constexpr bool rts_in_reverse = reverse == consume_in_reverse::legacy_half_reverse;
+    constexpr bool rts_in_reverse = reverse == consume_in_reverse::legacy_half_reverse || reverse == consume_in_reverse::yes;
```

Comment:
Please add a comment here saying that because range_tombstone_list deoverlaps range tombstones, we can just read them in reverse, even in full-reverse mode.

Reply:
Done in 935bb643ae